README.md

```diff
@@ -332,7 +332,28 @@ The IP configuration supports the following options:
     Manual DNS configuration can be specified via a list of addresses
     given in the `dns` option, a list of domains to search given in the
     `dns_search` option and a list of dns options to set given in the
-    `dns_options`.
+    `dns_options`. The list of available dns options is the same as
+    described in [man 5 resolv.conf](https://man7.org/linux/man-pages/man5/resolv.conf.5.html).
+    Currently, the list of supported dns options is:
+    - `attempts:n`, the number of times the resolver will send a query to its name servers before giving up and returning an error to the calling application.
+    - `debug`, this option enables the debug support.
+    - `edns0`, this option enables support for the DNS extensions described in [RFC 2671](https://tools.ietf.org/html/rfc2671).
+    - `ndots:n`, this options sets a threshold for the number of dots which must appear in a name given to [res_query()](https://man7.org/linux/man-pages/man3/res_query.3.html) before an initial absolute query will
+    be made.
+    - `no-check-names`, this option disables the modern BIND checking of incoming hostnames and mail names for invalid characters such as underscore (_), non-ASCII, or control characters.
+    - `no-reload`, disables automatic reloading of a changed configuration file.
+    - `no-tld-query`, attempt not to resolve an unqualified name as if it were a top level domain(TLD).
+    - `rotate`, round-robin selection of name servers from among those listed.
+    - `single-request`, this option makes glibc perform the IPv6 and IPv4 requests sequentially
+    - `single-request-reopen`, when this option is set, if two requests from the same port are not handled correctly it will close the socket and open a new one before sending the second request.
+    - `timeout:n`, the amount of time the resolver will wait for a response from a remote name server before
+    retrying the query via a different name server.
+    - `trust-ad`, this option controls the AD bit behavior of the stub resolver, if the `trust-ad` option has been set correctly, the applications can trust the AD bit in responses.
+    - `use-vc`, this option forces the use of TCP for DNS resolutions.
+
+**Note:** The "trust-ad" setting is only honored if the profile contributes name servers to resolv.conf,
```

Comment:
This should probably mention that DNS options are only supported for the NetworkManager provider

Reply:
this had been addressed at [Line336](https://github.com/linux-system-roles/network/pull/370/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R336)